modules/apps/jaggery/emm/src/emm/modules/backend-service-invoker.js

```diff
@@ -172,7 +172,7 @@ var backendServiceInvoker = function () {
         var options = new Array();
         if (IS_OAUTH_ENABLED) {
             var accessToken = getAccessToken();
-            if (!(!accessToken)) {
+            if (accessToken != null || accessToken != "") {
```

Comment:
we are using if (accessToken) in JS to check whether accessToken has being assigned with any type of explicit value except null, undefined, "" or false. Thus it's alright and much safe to use this way except in some special occasion.